src/main/java/duke/tasks/Todo.java

```diff
@@ -0,0 +1,17 @@
+package duke.tasks;
+
+public class Todo extends Task {
+
+    public Todo(String description) {
+        super(description);
+    }
+
+    @Override
+    public String toString() {
+        return "[T]" + super.toString();
+    }
+
+    public String storedTaskString() {
+        return "T" + "@" + super.storedTaskString();
+    }
+}
```

Comment:
Not sure I like the absence of javadocs here

Reply:
Have added the javadocs in recent commit!